### PR TITLE
Feat - Added Authentication Pages

### PR DESCRIPTION
**Pull request by SukanyaSen530**
- Added Log In page
- Added Sign Up Page
- Added Log Out Page

login - https://deploy-preview-6--weebo-action-figures.netlify.app/pages/login.html
logout - https://deploy-preview-6--weebo-action-figures.netlify.app/pages/logout.html
singup - https://deploy-preview-6--weebo-action-figures.netlify.app/pages/sign-up.html

### CODE REVIEW

File: pages/login.html [hunk 1]
```diff
@@ -22,24 +24,46 @@
...
+            </div>
+
+            <div class="input-group dark">
+                <label for="password" class="input-group-label">Password</label>
+                <input type="password" name="password" id="password" placeholder="Password" class="input-group-input">
+            </div>
+
+            <div class="input-small-group">
+                <label for="" class="input-small-label">Remember Me!</label>
+                <input type="checkbox" name="" id="" class="input-small">
```
[the remark this answers is not in the document]
Comment (SukanyaSen530) [reply pages/login.html:48]: Thanks for pointing it out, forgot to add the name and id

**Review by deepak29-git — APPROVED** [review]
Great work. clean code you can merge it

File: pages/login.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (SukanyaSen530) [reply pages/login.html]: fixed - a2e752a9c1bf35b2f1209eceb65c61f582218881

**Review by Mohitkumar33 — APPROVED** [review]
I think you can increase the font size of labels in your inputs.css file.

**Review by JowelTisso — APPROVED** [review]
Looks, good.
Suggestion: You can add more spacing in product card details and increase more width of img.

**Comment by SukanyaSen530** [comment]
@Mohitkumar33 Will Update it in my Component Library. :)

**Comment by SukanyaSen530** [comment]
@JowelTisso cool, will update in next PR

**Pull request merged**